### PR TITLE
Connect rails front end with the back end

### PR DESCRIPTION
**Pull request by ulisesedu7**
## Description
This PR connects the back end with the front end to display several messages 

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Checklist

- [X] My code follows the style guidelines of this project (no linters errors)
- [X] I have checked my code and corrected any misspellings
- [X] I have commented on my code
- [X] I have deleted any unused code as comments in my codebase

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
```
Comment (BertrandConxy) [inline README.md]: - [x] Good job 👏 But It looks like you have not used the [README template](https://github.com/microverseinc/readme-template) provided by Microverse. Currently, It contains content that doesn't look professional. Please use the README template provided in the [link here](https://github.com/microverseinc/readme-template) and then customize it to meet your project needs 😃 
⚠️ You will notice that It is an Updated template, a lot new sections and features has been added so please follow it all 😃

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+
```
Comment (BertrandConxy) [inline README.md]: - [x] In addition to what I suggested above concerning improvements on README, **It is also suggested to add a link to the project Rails API (backend) to help switch easily.**

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @ulisesedu7, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @ulisesedu7, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [x] Kindly as suggested previously update your backend app with the updated readme. You have only updated the frontend app readme.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**